MagicMaster/script.json

```diff
@@ -2,7 +2,7 @@
     "$schema": "https://github.com/DameryDad/roll20-api-scripts/blob/MagicMaster/MagicMaster/Script.json",
     "name": "MagicMaster",
     "script": "MagicMaster.js",
-    "version": "1.4.05",
+    "version": "1.4.06",
     "previousversions": ["2.044","2.045","2.046","2.048","3.051","1.3.00","1.3.01","1.3.02","1.3.03","1.4.01","1.4.02","1.4.04"],
```

Comment:
You should make sure to include the previous version of the script in this array

Reply:
This needs to go to match the other version releases, so RPGMaster is at v1.4.06 as a whole.

Reply:
OK, to make sure this goes through as quickly as possible I've added v1.4.05 back into the script.json previousversions. I will explore further how to make releases that both have bug fixes and new functionality, while stopping users loading the wrong versions (though I note the version selection drop-down disappeared from the Mod selection screen).